### PR TITLE
Added Selected Items label to multiple receivers for dialogs in Prepare > Databook sub menu

### LINKED ISSUE #8203: Showing number of selected items in multiple receiver
I suggest the multiple receiver to always show the number of selected items. I think it will be very helpful in helping users be keen on the items selected especially for sensitive operations like deleting and reshaping of data frames.
For example have a look at the dialog below.

### PR DESCRIPTION
**Pull request by derekagorhom**
fixes #8203 

I have added the Selected item label to `dlgdeletedataframe`, `dlgHideUnhidedataframe` and `dlgReorderdataframe`.
This PR is ready for review
@Patowhiz , @N-thony can you review this PR.

Thanks

### CODE REVIEW

**Comment by anastasia-mbithe** [comment]
@derekagorhom 
The feature works well. 
Though, do we really need it in the Reorder Data frame dialog? We ain't doing any selection there.
Additionally, the top, of the Hide and Unhide dialog as well as the Delete Data Frame dialog, looks so wordy and inconsistent in the (data frame & dataframe) labeling.
I would prefer having the number only( i.e Selected Data Frames: 2) it would look neater.
image

**Comment by derekagorhom** [comment]
I removed the selected item label from the reorder dialogue and also made minor changes to the other dialogues which should make them more consistent and less wordy now. 
@anastasia-mbithe can you have a look at it again. 

Thanks

File: instat/dlgHideDataframes.vb [hunk 1]
```diff
@@ -60,12 +61,20 @@ Public Class dlgHideDataframes
         ucrPnlHideUnhide.AddParameterValuesCondition(rdoHideDataFrame, "checked", "rdoHide")
         ucrPnlHideUnhide.AddParameterValuesCondition(rdoUnhideDataFrame, "checked", "rdoUnhide")
 
-        ucrReceiverMultiple.SetLinkedDisplayControl(lblHiddenDataFrames)
-        ucrReceiverMultipleUnhide.SetLinkedDisplayControl(lblUnhideDataFrame)
+        Dim lstControls As New List(Of Control)
+        lstControls.Add(lblHiddenDataFrames)
+        lstControls.Add(lblHiddenNumber)
+        ucrReceiverMultiple.SetLinkedDisplayControl(lstControls)
+        Dim LsControl As New List(Of Control)
```
Comment (N-thony) [inline instat/dlgHideDataframes.vb:68]: can you rename this variable correctly?

**Review by N-thony — CHANGES_REQUESTED** [review]

**Review by rdstern — APPROVED** [review]
Seems ok in the 2 dialogues for hide and delete.  So I am approving.
@Patowhiz did you also want it added to multiple receivers (like One Variable Summarise) where there are multiple variables?  Or were you thinking of it mainly for the delete data frames?

File: instat/dlgHideDataframes.vb [hunk 2]
```diff
@@ -60,12 +61,20 @@ Public Class dlgHideDataframes
         ucrPnlHideUnhide.AddParameterValuesCondition(rdoHideDataFrame, "checked", "rdoHide")
         ucrPnlHideUnhide.AddParameterValuesCondition(rdoUnhideDataFrame, "checked", "rdoUnhide")
 
-        ucrReceiverMultiple.SetLinkedDisplayControl(lblHiddenDataFrames)
-        ucrReceiverMultipleUnhide.SetLinkedDisplayControl(lblUnhideDataFrame)
+        Dim lstControls As New List(Of Control)
+        lstControls.Add(lblHiddenDataFrames)
+        lstControls.Add(lblHiddenNumber)
+        ucrReceiverMultiple.SetLinkedDisplayControl(lstControls)
+        Dim LstControl As New List(Of Control)
```
Comment (lloyddewit) [inline instat/dlgHideDataframes.vb:68]: Does not follow naming conventions. Rename to `lstControlsUnhidden` or similar?

Comment (derekagorhom) [reply instat/dlgHideDataframes.vb]: @lloyddewit i have made the corrections.

**Review by lloyddewit — APPROVED** [review]

**Comment by Patowhiz** [comment]
@rdstern @derekagorhom I was thinking of this to be added everywhere there is multiple variables selection. So this functionality to be inside the multiple receivers.
This is particularly important when users are working with many variables. Confirming the number of selected variables eases the use of the software.

**Comment by Patowhiz** [comment]
@derekagorhom if fixed as expected in the issue, changes made on the dialogs in this PR will have to be reverted.

**Comment by rdstern** [comment]
@derekagorhom I agree with the minor proviso that a few dialogues already have this feature - the Climatic > Tidy Daily Data is an example.  If the soltion applies generally then we should also check the code on these dialogues.

**Pull request merged**